### PR TITLE
Added new method in fil actor interface

### PR DESCRIPTION
**Pull request by sudo-shashank**
**Summary of changes**
Changes introduced in this pull request:
- Added `list_all_miners` method required for `Filecoin.StateListMiners`

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: fil_actor_interface/src/builtin/power/mod.rs [hunk 1]
```diff
@@ -94,6 +94,61 @@ impl State {
...
     }
 
+    /// Returns the addresses of every miner that has claimed power in the power actor
+    pub fn list_all_miners<BS: Blockstore>(self, store: &BS) -> anyhow::Result<Vec<Address>> {
+        match self {
+            State::V8(st) => {
+                let claims =
+                    fil_actors_shared::v8::make_map_with_root::<_, Claim>(&st.claims, store)?;
+                let mut miners = Vec::new();
+                claims.for_each(|bz, _claim| {
```
Comment (hanabi1224) [inline fil_actor_interface/src/builtin/power/mod.rs:104]: what is `bz`?

**Review by hanabi1224 — APPROVED** [review]

File: fil_actor_interface/src/builtin/power/mod.rs [hunk 2]
```diff
@@ -94,6 +94,61 @@ impl State {
         }
     }
 
+    /// Returns the addresses of every miner that has claimed power in the power actor
```
Comment (LesnyRumcajs) [inline fil_actor_interface/src/builtin/power/mod.rs:97]: Could we avoid repeating the code by using a macro? To my understanding, the code differs only with types, no?

**Review by LesnyRumcajs — APPROVED** [review]

File: fil_actor_interface/src/builtin/power/mod.rs (hunk 2 above)
Comment (sudo-shashank) [reply fil_actor_interface/src/builtin/power/mod.rs:98]: done

File: fil_actor_interface/src/builtin/power/mod.rs (hunk 1 above)
Comment (sudo-shashank) [reply fil_actor_interface/src/builtin/power/mod.rs]: used a better name `bytes`

**Pull request merged**